### PR TITLE
Fix typo and add tests for fonts install endpoint

### PR DESCRIPTION
**Pull request by matiasbenedetto**
## What?
- Fix a typo in code: See @anton-vlasenko comment here: https://github.com/WordPress/gutenberg/pull/52704#discussion_r1290722815
- Add tests to specify when the endpoint should fail with 400 (bad request) error.

## Testing Instructions
Run WP_REST_Fonts_Library_Controller_Test PHP unit tests

### CODE REVIEW

File: phpunit/fonts-library/class-wp-rest-fonts-library-controller-test.php
Comment (hellofromtonya) [inline phpunit/fonts-library/class-wp-rest-fonts-library-controller-test.php]:
```suggestion
			'with improper font face property' => array(
```

File: phpunit/fonts-library/class-wp-rest-fonts-library-controller-test.php
Comment (hellofromtonya) [inline phpunit/fonts-library/class-wp-rest-fonts-library-controller-test.php]:
```suggestion
		$this->assertSame( 400, $response->get_status(), 'Response status is not 400 when font face has both donwload_from_url and uploaded_file properties.' );
```

There's an Core initiative to replace all `assertEquals()` with `assertSame()`.

File: phpunit/fonts-library/class-wp-rest-fonts-library-controller-test.php
Comment (hellofromtonya) [inline phpunit/fonts-library/class-wp-rest-fonts-library-controller-test.php]: No empty lines here.
```suggestion
```

File: phpunit/fonts-library/class-wp-rest-fonts-library-controller-test.php
Comment (hellofromtonya) [inline phpunit/fonts-library/class-wp-rest-fonts-library-controller-test.php]:
```suggestion
	 * @param array $font_families Font families to install in theme.json format.
	 * @param array $files         Font files to install.
```

File: phpunit/fonts-library/class-wp-rest-fonts-library-controller-test.php
Comment (hellofromtonya) [inline phpunit/fonts-library/class-wp-rest-fonts-library-controller-test.php]:
```suggestion
			'not a font families array'       
 => array(
```

File: phpunit/fonts-library/class-wp-rest-fonts-library-controller-test.php
Comment (hellofromtonya) [inline phpunit/fonts-library/class-wp-rest-fonts-library-controller-test.php]:
```suggestion
			'empty array'                     
 => array(
```

File: phpunit/fonts-library/class-wp-rest-fonts-library-controller-test.php
Comment (hellofromtonya) [inline phpunit/fonts-library/class-wp-rest-fonts-library-controller-test.php]:
```suggestion
			'without slug'                    
 => array(
```

File: phpunit/fonts-library/class-wp-rest-fonts-library-controller-test.php
Comment (hellofromtonya) [inline phpunit/fonts-library/class-wp-rest-fonts-library-controller-test.php]:
```suggestion
			'with empty font face property'    => array(
```

**Review by hellofromtonya — APPROVED** [review]
Assuming I didn't again mess up the formatting for wpcs/phpcs ;), the changes LGTM 👍 Once the CI jobs are all 👍 , ready for merge. Great job @anton-vlasenko and @matiasbenedetto!

**Comment by hellofromtonya** [comment]
Well boo, seems I further messed up the alignment as WPCS is not happy. Instead of using the GH suggestion tool, I'll pull this PR down and fix it. Working on it now.

**Comment by matiasbenedetto** [comment]
Thanks for the review!

**Pull request merged**
